# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1489773609 -3600
#      Fri Mar 17 19:00:09 2017 +0100
# Node ID ecbd341564facea22d72f3e896f970d66c8bab62
# Parent  c1362a71fb0ebf9d894644758398a8d50ec2954d
Use Qt symbols already existing in Qt 4.6 (bug #50567)

* file-editor.cc (file_editor_tab_widget::mousePressEvent)
* webinfo.cc (webinfo_tab_bar::mousePressEvent):
  use Qt::MidButton instead of Qt::MiddleButton

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -2475,17 +2475,17 @@ file_editor::switch_tab (int direction, 
 // Functions of the the reimplemented tab widget
 //
 
 // Reimplement mouse event for filtering out the desired mouse clicks
 void
 file_editor_tab_widget::mousePressEvent(QMouseEvent *me)
 {
   if (me->type () != QEvent::MouseButtonDblClick &&
-       me->button() == Qt::MiddleButton)
+       me->button() == Qt::MidButton)
     {
       // Middle click into the tabbar -> close the tab
       for (int i = 0; i < tabBar ()->count (); i++)
         {
           QPoint clickPos = mapToGlobal (me->pos ());
           if (tabBar ()->tabRect (i).contains (tabBar ()->mapFromGlobal (clickPos)))
             {
               int idx = currentIndex ();
diff --git a/libgui/src/qtinfo/webinfo.cc b/libgui/src/qtinfo/webinfo.cc
--- a/libgui/src/qtinfo/webinfo.cc
+++ b/libgui/src/qtinfo/webinfo.cc
@@ -318,17 +318,17 @@ webinfo::pasteClipboard ()
 // Functions of the the reimplemented tab bar
 //
 
 // Reimplement mouse event for filtering out the desired mouse clicks
 void
 webinfo_tab_bar::mousePressEvent(QMouseEvent *me)
 {
   if (me->type () != QEvent::MouseButtonDblClick &&
-      me->button() == Qt::MiddleButton &&
+      me->button() == Qt::MidButton &&
       count () > 1)
     {
       // Middle click into the tabbar -> close the tab
       for (int i = 0; i < count (); i++)
         {
           QPoint clickPos = mapToGlobal (me->pos ());
           if (tabRect (i).contains (mapFromGlobal (clickPos)))
             {
